src/cogs/owner.py

```diff
@@ -619,7 +643,8 @@ def freeze_dict(d: dict[str, Any]) -> tuple[tuple[str, Any], ...]:
                 :tiling,
                 :text_type,
                 NULL,
-                ""
+                '',
+                ''
```

Comment:
What?

Reply:
sqlite is supposed to use single quotes for string literals, and a row was added for extra_frames

Reply:
Ah.